src/index.scss

```diff
@@ -8,6 +8,7 @@ $type-scales: (
     1: 18px,
 );
 $border-height: 3px;
+$popup-delay: 1s;
```

Comment:
Can we try reducing this to 0.25? I'm aiming to it just be enough that a use can move their mouse down and over it (as if they where going to the windows/mac task bar) without it opening, but fast enough that if they slow down or pause at all it feels like its opening? And the same should be for mouse off, if you mouse goes off and comes straight back on, it shouldn't start to close. Best advise I can give it to mirror the behavior in this regards (in terms of feel) from the old hud on http://play.getglimpse.com/.